v3/solonet/preview5/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+
+RUN apt-get update && apt-get install -y wget unzip
+WORKDIR /opt/neo-cli
+# TODO: replace with shell script for getting specified release tag
+RUN wget -q -O cli.zip https://github.com/neo-project/neo-node/releases/download/v3.0.0-preview5/neo-cli-linux-x64.zip
+RUN unzip -q -d /opt cli.zip && rm cli.zip
```

Comment:
Note that just for `neo-cli` I had to change the output dir to `/opt` instead of `/opt/neo-cli/`. What I otherwise would get is `/opt/neo-cli/neo-cli/`. It could be good to test this on your side as this might be a Linux vs Windows things, because comparing preview4 to preview5 the zip file seems to be constructed the same